mmdet3d/datasets/convert_utils.py

```diff
@@ -292,11 +292,10 @@ def get_kitti_style_2d_boxes(info: dict,
         # If mono3d=True, add 3D annotations in camera coordinates
         if mono3d and (repro_rec is not None):
             repro_rec['bbox_3d'] = np.concatenate(
-                [loc_3d, dim, rot],
-                axis=1).astype(np.float32).squeeze().tolist()
+                [loc, dim, rot], axis=1).astype(np.float32).squeeze().tolist()
```

Comment:
suggest adding some comments in the modification, e.g., here we use gravity center here?